### PR TITLE
Fix permissions for weaviate data directory.

### PR DESCRIPTION
**Pull request by dlorenc**
Fixes:

Related:

### CODE REVIEW

**Review by mattmoor — DISMISSED** [review]
How’d you find this?  Anything we can put in a test?

File: images/weaviate/configs/latest.apko.yaml [hunk 1]
```diff
@@ -9,6 +9,11 @@ paths:
     permissions: 0o777
     uid: 65532
     gid: 65532
+  - path: /var/lib/weaviate
+    type: directory
+    permissions: 0o777
```
Comment (luhring) [inline images/weaviate/configs/latest.apko.yaml:14]: not blocking, just curious why 777 is needed

**Review by luhring — APPROVED** [review]

**Comment by dlorenc** [comment]
> How’d you find this? Anything we can put in a test?

The tests pass now, I think they failed before but I can't actually find a run after the terraform stuff

**Comment by mattmoor** [comment]
They should run through terraform now.-M(obile)On Jul 1, 2023, at 2:30 PM, dlorenc ***@***.***> wrote:﻿

How’d you find this? Anything we can put in a test?

The tests pass now, I think they failed before but I can't actually find a run after the terraform stuff

File: images/weaviate/configs/latest.apko.yaml (hunk 1 above)
Comment (dlorenc) [reply images/weaviate/configs/latest.apko.yaml:14]: Fixed to 755

**Review by luhring — APPROVED** [review]

File: images/weaviate/configs/latest.apko.yaml (hunk 1 above)
Comment (anoncam) [reply images/weaviate/configs/latest.apko.yaml]: why does it need to be 0755?

What qualifies fox for this PR?

**Pull request merged**